lib/pg_eventstore/queries/event_type_queries.rb

```diff
@@ -33,6 +33,17 @@ def create_type(type)
       end.to_a.dig(0, 'id')
     end
 
+    # @param ids [Array<Integer>]
+    # @return [Array<Hash>]
+    def find_by_ids(ids)
+      return [] if ids.empty?
+
+      builder = SQLBuilder.new.from('event_types').where('id = ANY(?)', ids.uniq.sort)
```

Comment:
`ids.uniq.sort` why do you need a sort?

Reply:
You are right. There is no point in using `sort` here. It does not affect on the result anyhow